libs/gl-testing/Dockerfile

```diff
@@ -109,12 +106,17 @@ RUN apt-get update && apt-get install -qqy \
     python3-venv \
     nodejs \
     npm \
+    sudo \
     socat \
     unzip \
     wget \
     jq \
     && rm -rf /var/lib/apt/lists/*
 
+RUN groupadd -g $GID -o $DOCKER_USER &&\
+    useradd -m -u $UID -g $GID -G sudo -o -s /bin/bash $DOCKER_USER && \
+    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
```

Comment:
As commands in the docker container did not fail to me before, and because I can not find make recipes or commands that use sudo inside of the docker container, I am just wondering why we need to add the user to the sudoers here?

Reply:
I often end up wanting some more tools, for example `lnav` is very useful to drill down into the logs. Without `sudo` we can't install additional packages into the docker image, as frowned-upon as that may be, I really like having the ability to pull in more tools in the `docker-shell`.